cmd/experimental/kjobctl/go.sum

```diff
@@ -100,6 +100,8 @@ github.com/imdario/mergo v0.3.16 h1:wwQJbIsHYGMUyLSPrEq1CT16AhnhNJQ51+4fdHUnCl4=
 github.com/imdario/mergo v0.3.16/go.mod h1:WBLT9ZmE3lPoWsEzCh9LPo3TiwVN+ZKEjmz+hD27ysY=
 github.com/inconshreveable/mousetrap v1.1.0 h1:wN+x4NVGpMsO7ErUn/mUI3vEoE6Jt13X2s0bqwp9tc8=
 github.com/inconshreveable/mousetrap v1.1.0/go.mod h1:vpF70FUmC8bwa3OWnCshd2FqLfsEA9PFc4w1p2J65bw=
+github.com/jarcoal/httpmock v1.2.0 h1:gSvTxxFR/MEMfsGrvRbdfpRUMBStovlSRLw0Ep1bwwc=
+github.com/jarcoal/httpmock v1.2.0/go.mod h1:oCoTsnAz4+UoOUIf5lJOWV2QQIW5UoeUI6aM2YnWAZk=
```

Comment:
how do we have this with no go.mod updates?

Reply:
It's a very big question for me too :)

Reply:
Nothing.

Reply:
I added `rayutil` import. 

https://github.com/kubernetes-sigs/kueue/blob/2b225325a3521bf3df618265d9fd25524abab6e7/cmd/experimental/kjobctl/pkg/builder/ray_job_builder.go#L24

Inside this package using `github.com/jarcoal/httpmock`.

Reply:
Maybe because I'm using only one constant from this package.